public/app/plugins/datasource/graphite/datasource.ts

```diff
@@ -245,10 +245,11 @@ export function GraphiteDatasource(this: any, instanceSettings, $q, backendSrv,
     }
 
     const httpOptions: any = {
-      method: 'GET',
+      method: 'POST',
       url: '/metrics/find',
-      params: {
-        query: interpolatedQuery,
+      data: `query=${query}`,
```

Comment:
You need to use the `interpolatedQuery` here to support /metrics/find queries referencing template variables, like the dashboard here https://play.grafana.org/d/000000056/graphite-templated-nested?orgId=1